examples/S15_python/port.py

```diff
@@ -43,6 +43,6 @@
 portSurf.setAction('back', [spRed, spGreen], "reflect")
 
 # Ports
-testport = smoldyn.Port("testport", surface=portSurf, panel="front")
+testport = sim.addPort(name="testport", surface=portSurf, panel="front")
```

Comment:
Same as with the case of Path2d. `Simulation.addPort(a, b, c)` maps to `_smoldyn.Port(Simulation(), a, b, c)`.